.golangci.yml

```diff
@@ -84,7 +84,7 @@ linters-settings:
           - github.com/armon/go-metrics
 
           - cosmossdk.io/api
-          - cosmossdk.io/client/v2/autocli
+          - cosmossdk.io/client/v2
```

Comment:
The `depguard` definitions are all prefixes (unless they end with `$`). So this entry covers the import in app.go as well as a couple other places that need other things in that library.

And you can't just change an import since the stuff that's needed is in that specific package. I.e. changing the import in `app.go` from `cosmossdk.io/client/v2/autocli` to `cosmossdk.io/client/v2` would result in unresolved references.